contracts/lib/Organization.sol

```diff
@@ -260,4 +314,39 @@ contract Organization is OrganizationInterface, IsMemberInterface {
         isWorker_ = workers[_worker] > block.number;
     }
 
+
+    /* Private functions */
+
+    /**
+     * @notice Sets worker and its expiration block height.
+     *
+     * @param _worker Worker address to be added.
+     * @param _expirationHeight Expiration block height of worker.
+     *
+     * @return remainingBlocks_ Remaining number of blocks for which worker is
+     *                          active.
+     */
+    function setWorkerInternal(
+        address _worker,
+        uint256 _expirationHeight
+    )
+        private 
+        returns (uint256 remainingBlocks_)
+    {
+        require(
+            _worker != address(0),
+            "Worker address cannot be null."
+        );
+
+        require(
+            _expirationHeight > block.number,
+            "Expiration height must be in the future."
+        );
+
+        workers[_worker] = _expirationHeight;
+        remainingBlocks_ = _expirationHeight.sub(block.number);
```

Comment:
Let's not have this calculation in this function.

Reply:
Why?

Reply:
It costs gas, and if consumer of the function would need this information, it can check the expiration height of the worker (call to workers storage variable) and calculate it for that concrete case.

Reply:
tracked in https://github.com/OpenSTFoundation/mosaic-contracts/pull/547/files#r242095075